notebooks/MINDEvalResults.md

```diff
@@ -0,0 +1,140 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Offline Evaluation Metrics Visualizations
+This notebook visualizes user-specific performance metrics of various recommenders in the MIND validation dataset to assess effectiveness and ranking overlap. We explore two metric groups:
+1. **Effectiveness Metrics**: We use ranking-based metrics, Normalized Discounted Cumulative Gain (NDCG) and Mean Reciprocal Rank (MRR), to evaluate recommender effectiveness.
+2. **Ranking Overlap Metrics**: We use Rank-Based Overlap (RBO) to assess consistency in top-k recommendations relative to fianl rankings.
+
++++
+
+## 1. Setup
+
+```{code-cell} ipython3
+# Importing Libraries
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+from scipy.stats import sem, t
+from IPython.display import display, HTML
+
+# Loading Data
+mind_val_user_metrics = pd.read_csv('../outputs/mind-val-user-metrics.csv.gz')
+mind_val_user_metrics.head()
+```
```

Comment:
Style issue, but I would separate out imports from data load, and make the separate sub-sections with any descriptions in the notebook text instead of in code comments.